coalib/bearlib/abstractions/Linter.py

```diff
@@ -185,8 +185,12 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            basecheck_result = LocalBear.check_prerequisites()
+            if basecheck_result is not True:
+                return basecheck_result
+
             if shutil.which(cls.get_executable()) is None:
```

Comment:
already thought about this too. Will do that, though that interferes with https://github.com/coala/coala/issues/2073 a bit^^